manifests/config.pp

```diff
@@ -17,6 +17,12 @@
     'runoptions'        => $winlogbeat::run_options,
   })
 
+  if versioncmp($winlogbeat::real_version, '6') >= 0 {
+    $real_validate_cmd = "test config"
```

Comment:
suggest just calling this `validate_cmd`. You do not need to add `real_` since you are not "modifying" an existing `$validate_cmd` variable.